src/pages/costModels/costModel/sourcesToolbar.tsx

```diff
@@ -80,7 +80,7 @@ export const SourcesToolbar: React.SFC<SourcesToolbarProps> = ({
           </ToolbarItem>
         </ToolbarToggleGroup>
         <ToolbarItem>
-          <ReadOnlyTooltip isDisabled={actionButtonProps.isDisabled}>
+          <ReadOnlyTooltip isDisabled={true}>
```

Comment:
Do we want to change this, it may be used dynamically elsewhere?

Reply:
It appears that the `isDisabled` property is based on write permissions, so probably don't want to hard code this to always be disabled. I believe the intention is for `ReadOnlyTooltip` to omit the tooltip when a user doesn't have RBAC privileges.

Note that the `cost-demo` user has admin privileges in RBAC, so you may not see this change unless you try another user. We have multiple users to test different privileges (e.g., cost-cloud, cost-openshift, etc.).

Reply:
Oh sorry, I was testing something out and forgot to revert